tests/test_script_variant.py

```diff
@@ -7,21 +7,16 @@
 
 
 class ScriptVariantTests(unittest.TestCase):
-    def test_romanize_japanese_name_by_name_dict(self):
-        assert script_variant.romanize_japanese_name_by_name_dict(
+    def test_romanize_japanese_word(self):
+        assert script_variant.romanize_japanese_word(
```

Comment:
Add a test for a word which is in location dictionary, but not in name dictionary, to make sure we don't miss location dictionary. '中野' is not good because it's probably also in name dictionary.